BrainPortal/app/models/cluster_task.rb

```diff
@@ -2382,6 +2382,12 @@ def singularity_commands(command_script)
       sing_opts
     end
 
+    # (4b) Add tool config bindmounts, specified in 'overlay'
+    esc_local_bindmounts = bindmount_paths.inject("") do |sing_opts,(from_path,cont_path)|
```

Comment:
I'd replace `esc_local_bindmounts` with `esc_bindmounts` `esc_misc_local_bindmounts`, `esc_other_bindmounts`, `esc_config_bidmounts` ... to reduce confusion with `esc_local_dp_bindmounts`, not showstopper though

Reply:
You say "to reduce confusion with esc_local_dp_bindmounts".

But there is no `esc_local_dp_bindmounts` to be confused with.

There is an "esc_local_dp_mountpoints" which is not the same.

Reply:
never mind, those long names confused me